src/PowerShellEditorServices/Services/DebugAdapter/Debugging/VariableDetails.cs

```diff
@@ -356,10 +357,23 @@ private VariableDetails[] GetChildren(object obj, ILogger logger)
 
             return childVariables.ToArray();
         }
-
-        private static void AddDotNetProperties(object obj, List<VariableDetails> childVariables)
+        protected static void AddDotNetProperties(object obj, List<VariableDetails> childVariables, bool noRawView = false)
         {
             Type objectType = obj.GetType();
+
+            // For certain array or dictionary types, we want to hide additional properties under a "raw view" header
+            // to reduce noise. This is inspired by the C# vscode extension.
+            if (!noRawView &&
+                (
+                    obj is IEnumerable
+                    || obj is IDictionary
+                )
+            )
```

Comment:
This can just be `if (!noRawView && obj is IEnumerable)`

`IDictionary` implements `IEnumerable` so it'll never hit that branch and be true. Actually the compiler just doesn't emit the check, but still

Reply:
Good catch

Reply:
Implemented 95542b32d6d7439df98c604fe7673caf0852814f